.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,26 +2,17 @@
  Please fill this template unless your PR is very simple/straightforward.
  Do not forget to have a look at our Pull Request tutorial: http://www.azerothcore.org/wiki/Contribute#how-to-create-a-pull-request
 -->
+## 🍰 Pull request
+<!-- Describe the Pull request. What changes do you propose? -->
 
-<!-- WRITE A RELEVANT TITLE -->
 
-## Changes Proposed:
```

Comment:
Dont remove

Reply:
This section is unified under:

```
## 🍰 Pull request
<!-- Describe the Pull request. What changes do you propose? -->

[...]
```
A similar Issue like with the feature request - If you do a PR you ted to change things, thats the point of the PR - naturally a developer describes what he does in one paragraph.

Compare:

![image](https://user-images.githubusercontent.com/1238238/109404146-9e549a80-7963-11eb-9b39-cd07cb947e24.png)
![image](https://user-images.githubusercontent.com/1238238/109404152-ac0a2000-7963-11eb-84ad-4fd2ef2d3dad.png)

Reply:
Add back at lesst one bulletpoint underneathe the section so it is more clear <3

Reply:
I think both "Changes proposed" or "Pull request" can work fine, as long as it's clear to the user what he/she needs to write there

Reply:
> Add back at lesst one bulletpoint underneathe the section so it is more clear <3